File: android/app/src/main/java/mip/belllabs/moveinsaclay/MIS_SDK/SurveyManager.java
package mip.belllabs.moveinsaclay.MIS_SDK;

import android.view.View;
import android.webkit.WebSettings;
import android.webkit.WebView;
import android.webkit.WebViewClient;
import android.widget.Button;

public class SurveyManager {
    public static void loadSurvey (WebView webView){
        String url= "https://fr.surveymonkey.com/r/PRPGX98";
        webView.setWebViewClient(new WebViewClient()
        {
            @Override
            public boolean shouldOverrideUrlLoading(WebView view, String url)
            {
                view.loadUrl(url);
                return true;
            }
        });
        webView.loadUrl(url);
        WebSettings webSettings=webView.getSettings();
        webSettings.setJavaScriptEnabled(true);
    }
}

File: ios/MIP/SurveyManager.swift
import Foundation

class SurveyManager {
    static func loadSurvey (surveyWeb : UIWebView) {
        let myURL = URL(string: "https://fr.surveymonkey.com/r/PRPGX98")
        let myRequest = URLRequest(url: myURL!)
        surveyWeb.loadRequest(myRequest)
    }
}
